auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -112,6 +114,14 @@ public boolean awsNativeEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_CB_AWS_NATIVE);
     }
 
+    public boolean awsNativeDataLakeEnabled(String accountId) {
+        return isEntitlementRegistered(accountId, CDP_CB_AWS_NATIVE_DATALAKE);
+    }
+
+    public boolean awsNativeFreeIPAEnabled(String accountId) {
```

Comment:
Nit: Perhaps spelling it with camelcase (i.e. as `awsNativeFreeIpaEnabled`) would be more consistent with the rest of the code.